os/manual-rollbacks.md

```diff
@@ -2,6 +2,8 @@
 
 As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
 
+**Note**: Container Linux allows you to rollback to a previously active partition, or change to an older release channel (to downgrade to a stable version). It does not allow you to downgrade to an uninstalled, user defined version number.
```

Comment:
One possible issue is that saying rolling back to an arbitrary version is not supported sorta implies that the means of rolling back mentioned here are. They're not.

The only supported rollback is the "new partition won't boot, rollback before it boots" case (which happens automatically). Any time a version of Container Linux successfully boots, rolling back to a previous version (whether by channel downgrade or switching usr partitions), is no longer supported.